packages/corvid-cli/test/clone.e2e.js

```diff
@@ -401,4 +416,46 @@ describe("clone", () => {
       );
     });
   });
+
+  describe("Failed clone", () => {
+    it("should clean files", async () => {
+      const { tempDir, siteUrl } = await setupFailingClone();
+      const promise = clone({
+        url: siteUrl,
+        dir: tempDir
+      });
+      await expect(promise).rejects.toThrow(
+        getMessage("Pull_Client_Console_Fatal_Error")
+      );
```

Comment:
I think you should check that the clone actually wrote some files before you ensure it's empty.

Reply:
I think other tests cover it